fdm-app/app/components/blocks/indicators/aggregation-tree.tsx

```
@@ -0,0 +1,290 @@
+                                return (
+                                    <div
+                                        key={indId}
+                                        onClick={() => onIndicatorClick?.(indId)}
+                                        className={cn(
+                                            "flex items-center justify-between gap-3 p-2 rounded-md border border-dashed bg-card hover:bg-muted/40 transition-colors text-xs",
+                                            onIndicatorClick ? "cursor-pointer" : "",
+                                        )}
+                                        style={{
+                                            paddingLeft: `${Math.max(10, (depth + 1) * 16 + 10)}px`,
+                                        }}
+                                    >
```

Biome (2.4.16)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)